.github/workflows/job-postcommit-java-examples-dataflow-v2.yml

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+# Runs Java Examples on Dataflow runner v2 via cron and on push to master
+
+name: PostCommit Java Examples Dataflow v2
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+permissions: read-all
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          submodules: false
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+
+  postcommit-java-examples-dataflow-v2:
+    needs: set-properties
+    name: Run PostCommit Java Examples Dataflow v2
+    runs-on: [self-hosted, ubuntu-20.04]
+    strategy:
+      fail-fast: false
+      matrix: #JDK11 and 17 compilation will fail due to changes in ByteBuffer class since JDK9
+              #TODO: Retest once issue https://github.com/apache/beam/issues/23068 gets resolved.
+        java: ${{fromJson(needs.set-properties.outputs.properties).JavaTestProperties.SUPPORTED_VERSIONS}}
+        exclude:
+          - java: "11"
+          - java: "17"
```

Comment:
As mentioned in https://github.com/apache/beam/pull/23077#discussion_r988104603 we shouldn't be removing the Jenkins file if we have this constraint

Reply:
Oh, in the case of this workflow, the Java 8 version is executed successfully and thus, the only jenkins job removed is the JDK8 one. The jobs for [JDK11](https://github.com/fernando-wizeline/beam/blob/gam-postcommit-java-examples-dataflow-v2/.test-infra/jenkins/job_PostCommit_Java_Dataflow_Examples_Java11.groovy) and [JDK17](https://github.com/fernando-wizeline/beam/blob/gam-postcommit-java-examples-dataflow-v2/.test-infra/jenkins/job_PostCommit_Java_Examples_Dataflow_V2_Java17.groovy) remain here.

Reply:
Ah, my mistake!